features/support/env.rb

```diff
@@ -139,3 +144,13 @@ def extract_process_id(data)
   stdout = aruba.command_monitor.find(Aruba.platform.detect_ruby(last_command_started)).send(:stdout)
   expect(@service_current_process_id).not_to eq(extract_process_id(stdout))
 end
+
+Then(/^the box is "([^"]*)"$$/) do |box_name|
+  @box_name = box_name
+end
+
+Then(/^the binary for "([^"]*)" should be installed$/) do
+  binary_path = ENV['VAGRANT_HOME'] + '/data/service-manager/bin/docker/1.9.1/docker'
```

Comment:
hmm, the 1.9.1 seems very hard coded here. Could we not use the output of the install-command to make this dynamic or in some other form skip hard-coding the version?